clouddriver-eureka/README.md

```diff
@@ -41,4 +41,40 @@ Each account definition in AWS must also define a `discovery` field that denotes
 By default, only one Eureka is supported per AWS account. If you have multiple 
 `aws.accounts` configured in clouddriver which share an accountId, and wish to
 use a separate Eureka for each, then you can enable support by setting
-`eureka.provider.allowMultipleEurekaPerAccount` to true.
+`eureka.provider.allowMultipleEurekaPerAccount` to true. For example:
+
+    eureka:
+      provider:
+        allowMultipleEurekaPerAccount: true
+        enabled: true
+        accounts:
+          - name: my-ci-account
+            regions:
+              - us-west-2
+            readOnlyUrl: "http://10.0.0.1:8080/eureka/v2"
+          - name: my-qa-account
+            regions:
+              - us-west-2
+            readOnlyUrl: "http://10.0.0.2:8080/eureka/v2"
+
+    aws:
+      enabled: true
+      accounts:
+        - name: my-ci-account
+          accountId: 1234567899999
+          defaultKeyPair: 'my-keypair'
+          environment: ci
+          discovery: "http://10.0.0.1:8080/eureka/v2"
```

Comment:
No, I've been matching the eureka account name to the AWS account name. Did you want the AWS account config to have a separate eureka account name?